modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/EditRelationship.tsx

```diff
@@ -24,13 +24,30 @@ import {
 } from './ObjectRelationshipFormBase';
 import SelectRelationship from './SelectRelationship';
 
+interface EditRelationshipProps {
+	baseResourceURL: string;
+	deletionTypes: TDeletionType[];
+	hasUpdateObjectDefinitionPermission: boolean;
+	objectDefinitionExternalReferenceCode: string;
+	objectRelationship: ObjectRelationship;
+	parameterEndpoint: string;
+	parameterRequired: boolean;
+}
+
+export type TDeletionType = {
```

Comment:
@rebsilva Please, declare this type before the `EditRelationshipProps` interface to respect the precedence order of declaration and usage.